config.mk

```diff
@@ -19,9 +19,9 @@
 
 # Toolchain versions for SH
 sh_binutils_ver=2.34
-sh_gcc_ver=9.3.0
+sh_gcc_ver=10.1.0
```

Comment:
Thank you for your submission and your interest! 👍 
To be honest, I don't know if GCC 10 may be set by default, as this is preliminary. I think the best option here is to make a custom `config.mk` file like `config-gcc-10.mk`. What do you think?